miner/worker.go

```diff
@@ -197,6 +197,10 @@ type worker struct {
 	resubmitHook func(time.Duration, time.Duration) // Method to call upon updating resubmitting interval.
 }
 
+var (
+	IncomingMegaBundle = make(chan *types.MegaBundle)
```

Comment:
I'm really hesitant to use a global channel here.

Reply:
Why? Its done in other parts of code -

Reply:
Where else is there a global channel used in the mining worker? If so, I guess I'd be ok with this

Reply:
not under mining, iirc i saw ethclient code use a global chan as pattern (plus single channel is easier for miners to merge into theirs (i imagine touching fields around might be more chance of merge conflict)

Reply:
Alright, I guess I'm fine with it for now.